# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/storage/SessionStorageManager.h
# Commit: 44fb598d3ad8
# Full Hash: 44fb598d3ad83467805f960e973a6757c125ea5a
# Author: Yaron Tausky <ytausky@mozilla.com>
# Date: 2019-12-11 03:34:51
# Regressor Bug: 1593246
# File Overlap Count: 1
# Description:
#   Bug 1593246 - Part 3: Remove unused parameter from SessionStorageManager::GetSessionStorageCacheHelper r=dom-workers-and-storage-reviewers,sg
#   
#   Depends on D55659
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D55660
# ==============================================================================

diff -r c11ce752a12e -r 44fb598d3ad8 dom/storage/SessionStorageManager.h
--- a/dom/storage/SessionStorageManager.h	Tue Dec 10 12:12:32 2019 +0000
+++ b/dom/storage/SessionStorageManager.h	Mon Dec 09 19:38:34 2019 +0000
@@ -51,7 +51,6 @@
                      const nsACString& aOriginScope);
 
   nsresult GetSessionStorageCacheHelper(nsIPrincipal* aPrincipal,
-                                        nsIPrincipal* aStoragePrincipal,
                                         bool aMakeIfNeeded,
                                         SessionStorageCache* aCloneFrom,
                                         RefPtr<SessionStorageCache>* aRetVal);
